src/status_im2/contexts/share/view.cljs

```diff
@@ -0,0 +1,163 @@
+          :underlay-color   colors/neutral-80-opa-1-blur
+          :background-color :transparent
+          :on-press         #(copy-text-and-show-toast
+                              {:text-to-copy      profile-qr-url
+                               :post-copy-message (i18n/label :t/link-to-profile-copied)})
+          :on-long-press    #(copy-text-and-show-toast
+                              {:text-to-copy      profile-qr-url
+                               :post-copy-message (i18n/label :t/link-to-profile-copied)})}
+         [quo/text
+          {:style           (style/profile-address-content link-to-profile-max-width)
+           :size            :paragraph-1
+           :weight          :medium
+           :ellipsize-mode  :middle
+           :number-of-lines 1}
+          profile-qr-url]]]
+       [rn/view {:style style/share-button-container}
+        [quo/button
+         {:icon                true
+          :type                :blur-bg
+          :size                32
+          :accessibility-label :link-to-profile
+          :override-theme      :dark
+          :on-press            #(list-selection/open-share {:message profile-qr-url})}
+         :i/share]]]]
+
+     [rn/view {:style style/emoji-hash-container}
+      [rn/view {:style style/profile-address-container}
+       [rn/view {:style style/profile-address-column}
+        [quo/text
+         {:size   :paragraph-2
+          :weight :medium
+          :style  style/emoji-hash-label}
+         (i18n/label :t/emoji-hash)]
+        [rn/touchable-highlight
+         {:active-opacity   1
+          :underlay-color   colors/neutral-80-opa-1-blur
+          :background-color :transparent
+          :on-press         #(copy-text-and-show-toast
+                              {:text-to-copy      emoji-hash
+                               :post-copy-message (i18n/label :t/emoji-hash-copied)})
+          :on-long-press    #(copy-text-and-show-toast
+                              {:text-to-copy      emoji-hash
+                               :post-copy-message (i18n/label :t/emoji-hash-copied)})}
+         [rn/text {:style (style/emoji-hash-content emoji-hash-max-width)} emoji-hash]]]]
+      [rn/view {:style style/share-button-container}
+       [quo/button
+        {:icon                true
+         :type                :blur-bg
+         :size                32
+         :accessibility-label :link-to-profile
+         :override-theme      :dark
+         :style               {:margin-right 12}
+         :on-press            #(copy-text-and-show-toast
+                                {:text-to-copy      emoji-hash
+                                 :post-copy-message (i18n/label :t/emoji-hash-copied)})
+         :on-long-press       #(copy-text-and-show-toast
+                                {:text-to-copy      emoji-hash
+                                 :post-copy-message (i18n/label :t/emoji-hash-copied)})}
+        :i/copy]]]]))
+
+(defn wallet-tab
+  []
+  [:<>
```

Comment:
A lost fragment floating in space ;)

Reply:
@ilmotta : need this just to show that wallet tab UI is a WIP else I'd have to disable this tab.. what do you think?

Reply:
Oh if you mean the React fragment is necessary then of course keep it. I had the impression you could remove `[:<>]` without affecting the output.

Reply:
Oh no I misunderstood your lost fragment comment.
Yes that react fragment is un-necessary at the moment, I will get rid of it!

Reply:
resolved here : https://github.com/status-im/status-mobile/pull/15528/commits/7eb654b82e501eae9648b42a609500570a3d95db

Reply:
Sorry about the confusion, I'll try to be more explicit next time. Thanks